lib/MusicBrainz/Server/Controller/Doc.pm

```diff
@@ -26,24 +27,31 @@ sub show : Path('')
         return;
     }
 
-    my $bare = $c->req->param('bare') || 0;
-    $c->stash(
-        id => $id,
-        page => $page,
-        google_custom_search => DBDefs->GOOGLE_CUSTOM_SEARCH,
-    );
-
     if ($id =~ /^[^:]+:/i && $id !~ /^Category:/i) {
         $c->response->redirect(sprintf('http://%s/%s', DBDefs->WIKITRANS_SERVER, $id));
         $c->detach;
     }
 
+    my %props = (
+        useGoogleCustomSearch   => boolean_to_json(DBDefs->GOOGLE_CUSTOM_SEARCH),
+        id                      => $id,
+        page                    => $page,
+        wikiServer              => DBDefs->WIKITRANS_SERVER,
```

Comment:
Moving this to DBDefs.js rather than passing it as a prop would make sense to me

Reply:
Made that change. Not sure if this should use client or standard DBDefs, went for client since it doesn't need anything that isn't there.

Reply:
DBDefs-client is mostly only needed if the component is ever used in the browser, so you'd be fine to use the "server" DBDefs here. DBDefs-client isn't wrong either, but I suppose it might be confusing if someone expects it to be the server one since it's a server-only component.